style/icons/shield40_us_wi_rustic.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" overflow="visible" xmlns="http://www.w3.org/2000/svg">
+ <rect style="fill:#693f23;fill-rule:evenodd;stroke:#ffcd00;stroke-width:.919;stroke-linecap:square;stroke-opacity:1" width="19" height="19" x=".5" y=".5" ry="1"/>
+ <path d="m 9.1419483,17.714323 7.3188137,-2.05e-4 c -0.05033,-0.686531 0.118285,-1.237523 -0.289552,-1.831953 -0.04325,-0.629602 0.0026,-1.286432 0.237649,-1.934453 0.376457,-0.582832 -0.0937,-1.304761 0.3408,-1.886935 0.560756,-0.397092 -0.06193,-1.35413 0.554199,-1.876419 0.14588,-0.5630486 0.448743,-0.8872289 0.605546,-1.4149572 0.196139,-0.304771 0.185362,-0.4739592 0.308091,-0.8336734 -0.375729,-0.04222 -0.495427,0.4290981 -0.69504,0.600252 -0.180136,0.1564339 -0.313171,0.9766356 -0.385872,0.8000256 -0.561232,0.1547791 -0.679889,0.790991 -1.171548,1.121047 -0.333668,0.439714 -0.20919,-0.430125 -0.01455,-0.6910344 0.158939,-0.391661 0.32505,-0.6345065 0.675916,-0.7703741 C 16.9889,8.5585928 15.975678,8.4206091 16.542698,7.8888432 16.668832,7.3350235 15.823865,8.1533703 16.061411,7.4981562 16.198139,7.123392 16.343437,6.6718002 15.941471,6.3858415 15.654587,6.2622741 15.237117,6.3364047 15.448573,5.9637365 15.20402,5.6527204 14.510994,5.5133452 14.100801,5.5759633 13.454418,5.4708494 12.889927,4.9661124 12.186786,4.8965509 11.657983,4.6282005 10.797378,4.7506629 10.476561,4.2406632 9.9518073,3.8481206 9.5737933,3.2377489 8.5880835,3.6189296 8.4835003,3.2689477 9.3896029,2.3630331 8.6916732,2.2735844 8.3602311,2.4957851 8.062568,2.5703481 7.8000554,2.674756 7.2267039,2.8754461 6.5454074,3.3268485 5.9515216,3.0803909 5.6462778,2.734508 5.5146015,3.3555546 5.3300618,3.3274513 5.2042857,4.0277189 5.4430309,4.8582716 5.102562,5.4613536 4.6774022,5.5047117 4.1332572,5.8622638 3.9635215,6.3847182 3.6303627,6.5891903 3.7344435,7.1369477 4.1548442,7.1107689 4.5951765,7.6330686 4.0277253,7.8436763 4.0325874,8.4330855" style="fill:none;stroke:#fff;stroke-width:.75;stroke-linecap:butt;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none"/>
```

Comment:
The state shape could be simplified further. The Door Peninsula appears brighter than it should because of the more intricate coastline there. Try making Green Bay shorter.

Reply:
I tweaked the Door Peninsula and made the shield 1 pixel wider. It's made a huge difference in the legibility of 4-digit routes:

![Screenshot from 2022-03-30 15-45-50](https://user-images.githubusercontent.com/1732117/160918879-43778a37-ab9d-4a94-8011-1f302bab3a8c.png)
![Screenshot from 2022-03-30 15-46-02](https://user-images.githubusercontent.com/1732117/160918880-81212f30-2868-4a01-afd0-1be2e6af5b49.png)
![Screenshot from 2022-03-30 15-46-45](https://user-images.githubusercontent.com/1732117/160918881-056ce580-4c00-44b6-a9d1-75a2dd034141.png)
![Screenshot from 2022-03-30 15-47-05](https://user-images.githubusercontent.com/1732117/160918882-8f10df9f-5a85-42a6-b3b8-4aa76db94bd0.png)